po/oscap-anaconda-addon.pot

```diff
@@ -274,11 +274,6 @@ msgstr ""
 msgid "_Use SCAP Security Guide"
 msgstr ""
 
-#: tmp/oscap.glade.h:12
-msgid ""
-"No content found. Please enter data stream content or archive URI below:"
```

Comment:
it took me a few minutes to understand why this is being removed, but now I get it. :)

Before it was `URL` than changed to `URI` than changed back to `URL`

Reply:
A more elaborate explanation is part of the commit message.

Reply:
I saw that, but only after going through some `blaming`.